include/seqan3/search/fm_index/bi_fm_index.hpp

```diff
@@ -72,7 +72,12 @@ class bi_fm_index
     using sdsl_index_type = sdsl_index_type_;
 
     //!\brief The type of the underlying SDSL index for the reversed text.
-    using rev_sdsl_index_type = sdsl_index_type_;
+    using rev_sdsl_index_type = sdsl::csa_wt<sdsl_wt_index_type::wavelet_tree_type, // Wavelet tree type
```

Comment:
Does this type need to be "compatible" with the forward index type? Meaning that some template parameters have to be the same? If so we could maybe static_assert the respective requirements.

Reply:
Mhm, one question would be, if we allow stuff other than `csa_wt` - I don't think so. We could double check this.
Other than that the template params are taken from the forward index - sampling rate is fixed for the reverse index and the sampling type is also fixed. it doesn't really matter, because we will never ask the reverse index for location, so the sampling does not matter. Also, the sampling types are policy classes and not exposed in the SDSL, so I cant just do `sdsl_wt_index_type::sa_sampling_t`.